compiler/src/dotty/tools/dotc/util/Collections.scala

```diff
@@ -0,0 +1,39 @@
+/* NSC -- new Scala compiler
```

Comment:
I'm fine either way, apparently the original motivation for these methods is that they're faster than doing the equivalent using chains of methods from the standard library